.gitignore

```diff
@@ -1,5 +0,0 @@
-.env
```

Comment:
There is a reason why .env was ignored, so all your secrets will be kept safe. Now anyone can use your token and get access to the discord bot